tests/Doctrine/Tests/ORM/Functional/Ticket/GH10661/GH10661Test.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket\GH10661;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Tools\SchemaValidator;
+use Doctrine\Tests\OrmTestCase;
+
+/** @requires PHP >= 7.4 */
```

Comment:
Other occurrences of this in the project are multiline. Maybe old versions of PHPUnit ignore single line stuff. The next suspect will be `>=`

Reply:
Ah I can't push… push from maintainers is disabled. I'll open another PR I guess.

Reply:
Looks like that was the reason: https://github.com/doctrine/orm/pull/10946 :)